accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -2414,6 +2432,32 @@ Where `<keyword>` is `var` or `final` is treated like so:
 
 4.  Else the match failed. Execute the else `statement` if there is one.
 
+#### If-case element
+
+1.  Evaluate the `expression` producing `v`.
+
+2.  Match the `pattern` in the `caseHead` against `v`.
+
+3.  If the match succeeds:
+
+    1.  If there is a guard clause:
+
+        1.  Evaluate it. If it does not evaluate to a Boolean, throw a runtime
+            error. *This can happen if the guard expression's type is
+            `dynamic`.*
+
+        1.  If the guard evaluates to `true`, evaluate the then `element` and
+            yield the result into the collection.
+
+        2.  Else, evaluate the else `element` if there is one and yield the
+            result into the collection.
+
+    2.  Else there is no guard clause. Evaluate the then `element` and yield the
+        result into the collection.
+
+4.  Else the match failed. Else, evaluate the else `element` if there is one and
```

Comment:
nit: There might be a good reason for the current wording, but the repeated "Else" sounds odd.

Reply:
There is no good reason except for a copy/paste error.